dadb/src/main/kotlin/dadb/Dadb.kt

```diff
@@ -76,8 +76,8 @@ interface Dadb : AutoCloseable {
 
     @Throws(IOException::class)
     fun install(file: File, vararg options: String) {
-        if (supportsFeature("abb_exec")) {
-            abbExec("package", "install", "-S", file.length().toString(), *options).use { stream ->
+        if (supportsFeature("cmd")) {
+            execCmd("package", "install", "-S", file.length().toString(), *options).use { stream ->
```

Comment:
Should we keep the call to `abb_exec` in the event of `cmd` not being supported?

Reply:
abb_exec is a newer feature so cmd is guaranteed to be support on more devices